spec/lib/simp/cli/commands/config_spec_helper.rb

```diff
@@ -67,7 +67,7 @@ def generate_simp_lite_input_setting_values
     "no\n"                        << # don't set the GRUB password
     "no\n"                        << # don't use internet SIMP repos
     "no\n"                        << # SIMP is not LDAP server
-    "LOCAL\n"                     << # sssd domain
```

Comment:
The intent of the test using this generated input is to verify a user can input a domain.  The actual value doesn't matter, but it should not be empty.  If you don't want it to it to be confused with LOCAL, set it to something else (e.g., MYDOMAIN).

Reply:
With the latest changes it should not even be asking us about domains unless LDAP is true and SSSD is true.  So I think this is OBE.